openshift/Makefile

```diff
@@ -0,0 +1,11 @@
+# Get the directory where this Makefile is, so we can use it below for including
+DIR := $(strip $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
+include $(addprefix $(DIR)/vendor/github.com/openshift/build-machinery-go/make/, \
+    targets/openshift/deps.mk \
+)
+
+include $(DIR)/../.bingo/Variables.mk
+
+.PHONY: manifests
+manifests: $(KUSTOMIZE) $(YQ)
+	$(DIR)/generate-manifests.sh
```

Comment:
Please add a manifests verification that we can config for prow to include, like https://github.com/openshift/operator-framework-operator-controller/blob/f8bd9cea4551f467b1d621cbfc081d56f6baeace/openshift/Makefile#L14

Reply:
This was also added from the manifests merge, `verify-manifests` is in the Makefile now.